test/test_emitter_test.cpp

```diff
@@ -0,0 +1,14 @@
+#include "gtest/gtest.h"
+#include "goalc/emitter/CodeTester.h"
+#include "goalc/emitter/IGen.h"
+
+using namespace emitter;
+
+TEST(EmitterExperimentalTest, VF_NOP) {
+  CodeTester tester;
+	tester.init_code_buffer(1024);
+  tester.emit(IGen::nop_vf());
+
+	// ASK - easy way to get the opcode listing (that you know is correct) other than hand-crafting it?
```

Comment:
https://defuse.ca/online-x86-assembler.htm#disassembly and pick x64

I'd recommend just adding quick checks for each thing you add in `IGen`, getting the instruction slightly wrong is easy to do and sometimes it looks like it works.